templates/accounts/volunteer_profile.html

```diff
@@ -24,15 +24,15 @@ <h2 class="font-weight-bolder">Complete Your Profile</h2>
 
     <div class="row justify-content-center">
         <div class="col-md-8">
-
+            <p class="text-center text-danger mb-1 mt-3">{{ duplicate_error }}</p>
```

Comment:
The position of this element looks off. I think classes of `mb-3` and `mt-n2` will do fine.